I79 help documentation

pushyamig opened the pull request:
Fixes #79 and #110 

Preview of the documentation
https://pushyamig.github.io/canvas-course-manager-next/

pushyamig commented on docs/merge-sections.html:
```diff
@@ -3,5 +3,34 @@
+  <li>There are two panes in the Merge Sections tool. The right side pane shows sections already in the current course and will be updated to reflect new sections as you add them.</li>
+  <li>At the top of the left pane, enter the course name to merge in the search box.
+    <b>Tip:</b> When working with large numbers of lab/discussion sections, search results are most efficient when you include the course number (e.g. STATS 250, not just STATS). </li>
+  <li>Select the sections to merge by clicking on individual tiles or using the Select All box. <b>Tip:</b> Use the Sorting button to recording the sections to merge by enrollments or alphabetical order.</li>
```
```suggestion
  <li>Select the sections to merge by clicking on individual tiles or using the Select All box. <b>Tip:</b> Use the Sort button to re-order the sections to merge by enrollments or alphabetical order.</li>
```

melindakraft commented on docs/merge-sections.html:
```diff
@@ -3,5 +3,34 @@
+  <li>Launch Canvas Course Manager and click the Merge Sections button.</li>
+  <li>There are two panes in the Merge Sections tool. The right side pane shows sections already in the current course and will be updated to reflect new sections as you add them.</li>
+  <li>At the top of the left pane, enter the course name to merge in the search box.
+    <b>Tip:</b> When working with large numbers of lab/discussion sections, search results are most efficient when you include the course number (e.g. STATS 250, not just STATS). </li>
```
```suggestion
   <br> <b>Tip:</b> When working with large numbers of lab/discussion sections, search results are most efficient when you include the course number (e.g. STATS 250, not just STATS). </li>
```

melindakraft commented on docs/create-sections.html:
```diff
@@ -3,5 +3,22 @@
\ No newline at end of file
+<h1 class="mt-5">Create Sections</h1>
+<p>Some courses benefit from additional sections to organize students/participants for a variety of reasons. Course Manager makes new section creation quick and easy by uploading a simple CSV file.</p>
+<p><b>Tip: </b>Authorized users can add sections on the fly when add U-M or non-U-M users with this tool. See the help documentation for these features for more information.</p>
```
```suggestion
<p><b>Tip: </b>Authorized users can add sections on the fly when adding U-M or non-U-M users with this tool. See the help documentation for these features for more information.</p>
```

melindakraft commented on docs/create-sections.html:
```diff
@@ -3,5 +3,22 @@
+  <li>Root Account Administrator</li>
+  <li>Sub-Account Administrator</li>
+  <li>Support Consultant</li>
+  <li>Teachers and other course managers can request additional assistance by contacting <a href="mailto:4help@umich.edu">4help@umich.edu</a>.</li>
```
```suggestion
  <li>Teachers and other course managers can request assistance by contacting <a href="mailto:4help@umich.edu">4help@umich.edu</a>.</li>
```

melindakraft commented on docs/add-um-users.html:
```diff
@@ -3,5 +3,41 @@
+  <li>Drag and drop, or click the link to upload your saved CSV file.</li>
+  <li>CSV file data is presented back to you. Verify the results and click Submit.</li>
+</ul>
+<h5>How to Add U-M Users to Multiple Sections?</h5>
```
```suggestion
<h5>How to Add U-M Users to Multiple Sections</h5>
```

melindakraft replied:
```suggestion
<h5>How to Add U-M Users to Multiple Sections</h5>
```

melindakraft commented on docs/add-um-users.html:
```diff
@@ -3,5 +3,41 @@
+  <li>CSV file data is presented back to you. Verify the results and click Submit.</li>
+</ul>
+<h5>How to Add U-M Users to Multiple Sections?</h5>
+<p>This feature allows administrators to add U-M users to more than one existing section at the same time. New sections cannot be created with this tool.</p>
```
```suggestion
<p>This feature allows administrators to add U-M users to more than one existing section at the same time. Users are not required to have the same role. New sections cannot be created with this tool.</p>
```

melindakraft commented on docs/add-um-users.html:
```diff
@@ -3,5 +3,41 @@
+</ul>
+<h5>How to Add U-M Users to Multiple Sections?</h5>
+<p>This feature allows administrators to add U-M users to more than one existing section at the same time. New sections cannot be created with this tool.</p>
+<p>To use this feature you must include the Canvas section ID (not the section SIS ID) in your CSV file. A list of course section ID is included with this tool to help with CSV file preparation. </p>
```
```suggestion
<p>To use this feature you must include the Canvas section ID (not the section SIS ID) in your CSV file. A list of course section IDs is included with this tool to help with CSV file preparation. </p>
```

melindakraft commented on docs/add-non-um-users.html:
```diff
@@ -3,5 +3,34 @@
-<p>Follow documentation for this Feature</p>
\ No newline at end of file
+<h1 class="mt-5">Add Non-UM Users</h1>
+<p>External users (those without a U-M uniqname) can be added Canvas courses with the Course Manager tool. New external users who have never accessed Canvas will receive information by email to help them select a login method.  by course managers and administrators.</p>
```
```suggestion
<p>External users (those without a U-M uniqname) can be added to Canvas courses with the Course Manager tool. New external users who have never accessed Canvas will receive information by email to help them select a login method. </p>
```

melindakraft commented on docs/add-non-um-users.html:
```diff
@@ -3,5 +3,34 @@
-<h1 class="mt-5">Add Non-UM Users Documentation</h1>
-<p>Follow documentation for this Feature</p>
\ No newline at end of file
+<h1 class="mt-5">Add Non-UM Users</h1>
```
```suggestion
<h2 class="mt-5">Add Non-UM Users</h2>
```

melindakraft commented on docs/add-non-um-users.html:
```diff
@@ -3,5 +3,34 @@
\ No newline at end of file
+<h1 class="mt-5">Add Non-UM Users</h1>
+<p>External users (those without a U-M uniqname) can be added Canvas courses with the Course Manager tool. New external users who have never accessed Canvas will receive information by email to help them select a login method.  by course managers and administrators.</p>
+<h5>Who Can Add Non-UM Users?</h5>
```
```suggestion
<h2>Who Can Add Non-UM Users</h2>
```

melindakraft commented on docs/add-non-um-users.html:
```diff
@@ -3,5 +3,34 @@
+  <li>Support Consultant</li>
+  <li>Teachers and other course managers who normally have access to the People tool</li>
+</ul>
+<h5>How to Add Non-UM Users?</h5>
```
```suggestion
<h2>How to Add Non-UM Users</h2>
```

melindakraft commented on docs/add-non-um-users.html:
```diff
@@ -3,5 +3,34 @@
+</ul>
+<h5>How to Add Non-UM Users?</h5>
+<p>External users can be added one at time by typing in the user’s email address, or in bulk using a CSV file.</p>
+<h5>Adding External Users One at a Time</h5>
```
```suggestion
<h2>Adding External Users One at a Time</h2>
```

ssciolla replied:
I'd make this an `h6`, since that's what the level above is.

melindakraft commented on docs/add-non-um-users.html:
```diff
@@ -3,5 +3,34 @@
+<h5>How to Add Non-UM Users?</h5>
+<p>External users can be added one at time by typing in the user’s email address, or in bulk using a CSV file.</p>
+<h5>Adding External Users One at a Time</h5>
+<ul class="list-group-numbered">
```
```suggestion
<ol>
```

melindakraft commented on docs/add-non-um-users.html:
```diff
@@ -3,5 +3,34 @@
+  <li>Choose a user role from the dropdown menu.</li>
+  <li>Select a section to add the user to by clicking on one of the section name tiles. User can only be added one section with this tool.</li>
+  <li>Click Submit.</li>
+</ul>
```
```suggestion
</ol>
```

melindakraft commented on docs/add-non-um-users.html:
```diff
@@ -3,5 +3,34 @@
+  <li>Select a section to add the user to by clicking on one of the section name tiles. User can only be added one section with this tool.</li>
+  <li>Click Submit.</li>
+</ul>
+<h5>Adding Multiple External Users to a Single Section with a CSV File</h5>
```
```suggestion
<h2>Adding Multiple External Users to a Single Section with a CSV File</h2>
```

melindakraft commented on docs/add-non-um-users.html:
```diff
@@ -3,5 +3,34 @@
+  <li>Click Submit.</li>
+</ul>
+<h5>Adding Multiple External Users to a Single Section with a CSV File</h5>
+<ul class="list-group-numbered">
```
```suggestion
<ol>
```

melindakraft commented on docs/index.html:
```diff
@@ -3,8 +3,23 @@
+<h3>Tool Authorization</h3>
+<p>Like all third party tools, the first time you access Course Manager you will be prompted to authorize the tool. You can revoke your authorization in your Canvas Account settings. Updates to the Course Manager code may require you to reauthorize the tool. 
+</p>
+<h3>Renaming Your Course</h3>
```
```suggestion
<h2>Renaming Your Course</h2>
```

melindakraft commented on docs/index.html:
```diff
@@ -3,8 +3,23 @@
\ No newline at end of file
+<p>Canvas Course Manager (CCM) is a tool developed by University of Michigan Teaching and Learning that helps instructors, designers, TAs, and administrators perform some course management tasks quickly and easily. </p>
+<p>Only U-M uniqname holders can use CCM.</p>
+<h3>Tool Authorization</h3>
```
```suggestion
<h2>Tool Authorization</h2>
```

melindakraft commented on docs/index.html:
```diff
@@ -3,8 +3,23 @@
+<h3>Renaming Your Course</h3>
+<p>Renaming a course is helpful when multiple sections are combined (merged) into a single Canvas course site. </p>
+<h5>Who Can Use This Feature?</h5>
+<ul class="mx-3">
```
```suggestion
<ol>```

melindakraft commented on docs/index.html:
```diff
@@ -3,8 +3,23 @@
+  <li>Authorized user can rename a course from the Canvas Course Manager main page.</li>
+  <li>Launch the Canvas Course Manager from the course navigation menu. If you are already in the tool, click the Canvas Course Manager breadcrumb link at the top of the screen.</li>
+  <li>Click the Edit pencil icon next to the Course name, edit the course name as desired and click Save.</li>
+</ul>
```
```suggestion
</ol>
```

melindakraft commented on docs/index.html:
```diff
@@ -3,8 +3,23 @@
+</p>
+<h3>Renaming Your Course</h3>
+<p>Renaming a course is helpful when multiple sections are combined (merged) into a single Canvas course site. </p>
+<h5>Who Can Use This Feature?</h5>
```
```suggestion
<h3>Who Can Use This Feature</h3>
```

melindakraft commented on docs/index.html:
```diff
@@ -3,8 +3,23 @@
+  <li>Sub-Account Administrator.</li>
+  <li>Support Consultant.</li>
+  <li>All other roles should contact <a href="mailto:4help@umich.edu">4help@umich.edu</a> for assistance.</li>
+</ul>
```
```suggestion
</ol>
```

melindakraft commented on docs/merge-sections.html:
```diff
@@ -3,5 +3,34 @@
+  <li>Sub-account administrators can merge any sections in their assigned subaccount(s) only.</li>
+  <li>Teachers can merge sections only where they have the Teacher role. </li>
+</ul>
+<h5>How to Merge Sections</h5>
```
```suggestion
<h2>How to Merge Sections</h2>
```

melindakraft commented on docs/merge-sections.html:
```diff
@@ -3,5 +3,34 @@
+  <li>Teachers can merge sections only where they have the Teacher role. </li>
+</ul>
+<h5>How to Merge Sections</h5>
+<ul class="list-group-numbered">
```
```suggestion
<ol>
```

melindakraft commented on docs/merge-sections.html:
```diff
@@ -3,5 +3,34 @@
+  <li>Select the sections to merge by clicking on individual tiles or using the Select All box. <b>Tip:</b> Use the Sort button to re-order the sections to merge by enrollments or alphabetical order.</li>
+  <li>To remove sections from the right side pane, click the desired section tile, then click the undo button.</li>
+  <li>When you are ready to merge the sections, click the Go Merge button.</li>
+</ul>
```
```suggestion
</ol>
```

melindakraft commented on docs/create-sections.html:
```diff
@@ -3,5 +3,22 @@
+<h1 class="mt-5">Create Sections</h1>
+<p>Some courses benefit from additional sections to organize students/participants for a variety of reasons. Course Manager makes new section creation quick and easy by uploading a simple CSV file.</p>
+<p><b>Tip: </b>Authorized users can add sections on the fly when add U-M or non-U-M users with this tool. See the help documentation for these features for more information.</p>
+<h5>Who Can Create Sections?</h5>
```
```suggestion
<h2>Who Can Create Sections</h2>
```

melindakraft commented on docs/create-sections.html:
```diff
@@ -3,5 +3,22 @@
+  <li>Support Consultant</li>
+  <li>Teachers and other course managers can request additional assistance by contacting <a href="mailto:4help@umich.edu">4help@umich.edu</a>.</li>
+</ul>
+<h5>How to Create Sections?</h5>
```
```suggestion
<h2>How to Create Sections?</h2>
```

melindakraft commented on docs/create-sections.html:
```diff
@@ -3,5 +3,22 @@
+  <li>Teachers and other course managers can request additional assistance by contacting <a href="mailto:4help@umich.edu">4help@umich.edu</a>.</li>
+</ul>
+<h5>How to Create Sections?</h5>
+<ul class="list-group-numbered">
```
```suggestion
<ol>
```

melindakraft commented on docs/create-sections.html:
```diff
@@ -3,5 +3,22 @@
+  <li>Prepare and save a CSV file for upload (you can download an example file).</li>
+  <li>When your file is saved, drag and drop the file, or click the link Browse Your Files.</li>
+  <li>Your file will display on the screen. Verify the results and click Submit. Click Cancel to Abort.</li>
+</ul>
```
```suggestion
</ol>
```

melindakraft commented on docs/add-um-users.html:
```diff
@@ -3,5 +3,41 @@
+<p><b>Tip: </b>Instructors and other roles with People tool access can add U-M users in bulk by entering a list of email addresses or Uniqunames (login ID) in the People tool, being aware that the users may receive a new course invitation message from Canvas.</p>
+<h5>How to Add U-M Users to a Single Section?</h5>
+<p>This feature allows administrators to select an existing section or create a new section to accept the users.</p>
+<ul class="list-group-numbered">
```
```suggestion
<ol>
```

melindakraft commented on docs/add-um-users.html:
```diff
@@ -3,5 +3,41 @@
+  <li>Prepare and save your CSV file following the onscreen instructions for header and row content.</li>
+  <li>Drag and drop, or click the link to upload your saved CSV file.</li>
+  <li>CSV file data is presented back to you. Verify the results and click Submit.</li>
+</ul>
```
```suggestion
</ol>
```

ssciolla reviewed (commented):
Working my way through this. I'm releasing some initial thoughts and text change suggestions.

ssciolla commented:
Lot of comments out there already. @pushyamig, I will let you work through them first. Happy to review again for grammar, clarity, HTML structure after we've resolved some.

melindakraft commented on docs/gradebook-canvas.html:
```diff
@@ -3,5 +3,19 @@
-<h1 class="mt-5">Format Canvas Gradebook Documentation</h1>
-<p>Follow documentation for this Feature</p>
+<h1 class="mt-5">Format Canvas Gradebook</h1>
+<p>This Feature allows you to format exported Canvas Gradebook CSV file for uploading into Faculty Center’s Grade Roster. </p>
```
```suggestion
<p>This feature creates a CSV formatted for upload to Faculty Center. It requires a completed exported (downloaded) Canvas gradebook CSV file. </p>
```

pushyamig commented:
> Lot of comments out there already. @pushyamig, I will let you work through them first. Happy to review again for grammar, clarity, HTML structure after we've resolved some.

I will work on it in an hour or so.

melindakraft commented on docs/gradebook-canvas.html:
```diff
@@ -3,5 +3,19 @@
-<p>Follow documentation for this Feature</p>
+<h1 class="mt-5">Format Canvas Gradebook</h1>
+<p>This Feature allows you to format exported Canvas Gradebook CSV file for uploading into Faculty Center’s Grade Roster. </p>
+<h5>Who Can use Format Canvas Gradebook?</h5>
```
```suggestion
<h2>Who Can use Format Canvas Gradebook?</h2>
```

ssciolla replied:
@pushyamig, @melindakraft is probably right about the heading levels, it's better for accessibility not to skip a bunch of them. If this is for style, you can use the `h2` and then apply an `"h5"` classname, I think. See https://getbootstrap.com/docs/4.0/content/typography/#headings

melindakraft commented on docs/gradebook-canvas.html:
```diff
@@ -3,5 +3,19 @@
+  <li>Support Consultant</li>
+  <li>Teachers and other course managers who normally have access to the People tool</li>
+</ul>
+<h5>How to Use Format Canvas Gradebook?</h5>
```
```suggestion
<h2>How to Use Format Canvas Gradebook?</h2>
```

melindakraft commented on docs/gradebook-canvas.html:
```diff
@@ -3,5 +3,19 @@
+  <li>Teachers and other course managers who normally have access to the People tool</li>
+</ul>
+<h5>How to Use Format Canvas Gradebook?</h5>
+<ul class="list-group-numbered">
```
```suggestion
<ol>
```

melindakraft commented on docs/gradebook-canvas.html:
```diff
@@ -3,5 +3,19 @@
+  <li>Follow the instructions from the Requirement section on the “Upload your CSV File” page.</li>
+  <li>Drag and drop, or click the link to upload your saved CSV file.</li>
+  <li>The tool will convert your file to an MPathways compatible format and download it back to your computer.</li>
+</ul>
```
```suggestion
</ol>
```

melindakraft commented on docs/gradebook-canvas.html:
```diff
@@ -3,5 +3,19 @@
+<ul class="list-group-numbered">
+  <li>Click the Format Canvas Gradebook button in Canvas Course Manager.</li>
+  <li>Follow the instructions from the Requirement section on the “Upload your CSV File” page.</li>
+  <li>Drag and drop, or click the link to upload your saved CSV file.</li>
```
```suggestion
  <li>Drag and drop, or click the link to upload your exported gradebook CSV file.</li>
```

melindakraft commented on docs/gradebook-canvas.html:
```diff
@@ -3,5 +3,19 @@
+  <li>Click the Format Canvas Gradebook button in Canvas Course Manager.</li>
+  <li>Follow the instructions from the Requirement section on the “Upload your CSV File” page.</li>
+  <li>Drag and drop, or click the link to upload your saved CSV file.</li>
+  <li>The tool will convert your file to an MPathways compatible format and download it back to your computer.</li>
```
```suggestion
  <li>The tool will convert your file to a Faculty Center compatible format and download it back to your computer.</li>
```

melindakraft commented on docs/gradebook-thirdparty.html:
```diff
@@ -3,5 +3,23 @@
-<h1 class="mt-5">Format Third-Party Gradebook Documentation</h1>
-<p>Follow documentation for this Feature</p>
+<h1 class="mt-5">Format Third-Party Gradebook</h1>
+<p>This feature allows you to Format a CSV file exported from an external tool for importing into the Canvas Gradebook. This tool creates a new version of the uploaded file so it only includes students in the selected sections and is formatted appropriately for uploading to Canvas gradebook using the
```
```suggestion
<p>This feature creates a formatted CSV file from third party tools that are not integrated with Canvas that can be used to import grades into the Canvas Gradebook. This tool can be used to trim the third party data to selected course sections. To import the new CSV to the gradebook, see the 
```

melindakraft commented on docs/gradebook-thirdparty.html:
```diff
@@ -3,5 +3,23 @@
+<p>This feature allows you to Format a CSV file exported from an external tool for importing into the Canvas Gradebook. This tool creates a new version of the uploaded file so it only includes students in the selected sections and is formatted appropriately for uploading to Canvas gradebook using the
+  <a href="https://community.canvaslms.com/docs/DOC-12786-415255013" target="_blank">Canvas gradebook import function.</a>.
+</p>
+<h5>Who Can Use Format Third-Party Gradebook?</h5>
```
```suggestion
<h2>Who Can Use Format Third-Party Gradebook</h2>
```

melindakraft commented on docs/gradebook-thirdparty.html:
```diff
@@ -3,5 +3,23 @@
+  <li>Support Consultant</li>
+  <li>Teachers and other course managers who normally have access to the People tool</li>
+</ul>
+<h5>How to Use Format Third-Party Gradebook?</h5>
```
```suggestion
<h2>How to Use Format Third-Party Gradebook</h2>
```

melindakraft commented on docs/gradebook-thirdparty.html:
```diff
@@ -3,5 +3,23 @@
+  <li>Teachers and other course managers who normally have access to the People tool</li>
+</ul>
+<h5>How to Use Format Third-Party Gradebook?</h5>
+<ul class="list-group-numbered">
```
```suggestion
<ol>
```

melindakraft commented on docs/gradebook-thirdparty.html:
```diff
@@ -3,5 +3,23 @@
+<h5>How to Use Format Third-Party Gradebook?</h5>
+<ul class="list-group-numbered">
+  <li>Click the Format Third‑Party Gradebook button in Canvas Course Manager.</li>
+  <li>Select one or more sections whose students you want to collect grades from your CSV for.</li>
```
```suggestion
  <li>Select one or more sections that contain the students you want in the generated grade import CSV file.</li>
```

melindakraft commented on docs/gradebook-thirdparty.html:
```diff
@@ -3,5 +3,23 @@
+  <li>Download the example CSV file, if needed.</li>
+  <li>Prepare and save your CSV file following the onscreen instructions for header and row content.</li>
+  <li>Drag and drop, or click the link to upload your saved CSV file.</li>
+  <li>The tool will create a grades CSV file that contains only the data for designated course sections and download it back to your computer.</li>
```
```suggestion
  <li>The tool will create a grades CSV file that contains only the data for designated course sections and download it back to your computer.</li>
  <li>Review the generated file and <a href="https://community.canvaslms.com/docs/DOC-12786-415255013" target="_blank">import the grades into your Canvas gradebook.</a>.</li>
```

melindakraft commented on docs/gradebook-thirdparty.html:
```diff
@@ -3,5 +3,23 @@
+  <li>Prepare and save your CSV file following the onscreen instructions for header and row content.</li>
+  <li>Drag and drop, or click the link to upload your saved CSV file.</li>
+  <li>The tool will create a grades CSV file that contains only the data for designated course sections and download it back to your computer.</li>
+</ul>
```
```suggestion
</ol>
```

pushyamig commented on docs/add-non-um-users.html:
```diff
@@ -3,5 +3,34 @@
+<p>External users can be added one at time by typing in the user’s email address, or in bulk using a CSV file.</p>
+<h5>Adding External Users One at a Time</h5>
+<ul class="list-group-numbered">
+  <li>Click the Add Non U-M Users button in Canvas Course Manager.</li>
```
OK, I made it Canvas Course Manager instead of Course manage on L7 and I change to `on landing page`

pushyamig commented on docs/add-non-um-users.html:
```diff
@@ -3,5 +3,34 @@
+<ul class="list-group-numbered">
+  <li>Click the Add Non U-M Users button in Canvas Course Manager.</li>
+  <li>Choose the “Add One User Manually” option and click Select.</li>
+  <li>Enter the user’s email address and click Search.</li>
```
Done

pushyamig commented on docs/add-non-um-users.html:
```diff
@@ -3,5 +3,34 @@
+</ul>
+<h5>Adding Multiple External Users to a Single Section with a CSV File</h5>
+<ul class="list-group-numbered">
+  <li>Click the Add Non U-M Users button in Canvas Course Manager.</li>
```
Done

pushyamig commented on docs/add-non-um-users.html:
```diff
@@ -3,5 +3,34 @@
-<p>Follow documentation for this Feature</p>
\ No newline at end of file
+<h1 class="mt-5">Add Non-UM Users</h1>
+<p>External users (those without a U-M uniqname) can be added Canvas courses with the Course Manager tool. New external users who have never accessed Canvas will receive information by email to help them select a login method.  by course managers and administrators.</p>
```
Done! Thanks this was useful.

pushyamig commented on docs/add-um-users.html:
```diff
@@ -3,5 +3,41 @@
+  <li>Root Account Administrator</li>
+  <li>Sub-Account Administrator</li>
+  <li>Support Consultant</li>
+  <li>Teachers and other course managers can request additional assistance by contacting <a href="mailto:4help@umich.edu">4help@umich.edu</a>.</li>
```
I am not sure on that. @melindakraft initial [doc](https://docs.google.com/document/d/15xDvkqgjKq_0xZQbJraD6XON1DXoXzSyMrR8vHvSaD8/edit#heading=h.wb3yhhrgqotc) added them on as list. So I just did it like that. I don't have a huge preference but it is @melindakraft call

pushyamig commented:
I resolved all the suggestion If I missing some of them sorry since lot of comments. Please look for latest and suggest as needed.

melindakraft commented on docs/index.html:
```diff
@@ -3,8 +3,25 @@
+</ul>
+<h3 class="h6">How to Rename A Course</h3>
+<ol>
+  <li>Authorized user can rename a course from the on landing page.</li>
```
```suggestion
  <li>Administrators can rename a course from the Course Manager landing page.</li>
```

melindakraft commented on docs/index.html:
```diff
@@ -3,8 +3,25 @@
+<h3 class="h6">How to Rename A Course</h3>
+<ol>
+  <li>Authorized user can rename a course from the on landing page.</li>
+  <li>Launch the Canvas Course Manager from the course navigation menu. If you are already in the tool, click the Canvas Course Manager breadcrumb link at the top of the screen.</li>
```
```suggestion
  <li>Launch the Canvas Course Manager from the course navigation menu. <br><b>Tip: </b>If you are already in the tool, click the Canvas Course Manager breadcrumb link at the top of the screen to go to the landing page.</li>
```

melindakraft commented on docs/merge-sections.html:
```diff
@@ -3,5 +3,38 @@
+  <li>To remove sections from the right side pane, click the desired section tile, then click the undo button.</li>
+  <li>When you are ready to merge the sections, click the Go Merge button.</li>
+  </ol>
+<h6>More Tips for using Merge Sections</h6>
```
```suggestion
<h2>More Tips for using Merge Sections</h2>
```

melindakraft commented on docs/merge-sections.html:
```diff
@@ -3,5 +3,38 @@
+  <li>When you are ready to merge the sections, click the Go Merge button.</li>
+  </ol>
+<h6>More Tips for using Merge Sections</h6>
+<ol>
```
```suggestion
<ul>
```

melindakraft commented on docs/merge-sections.html:
```diff
@@ -3,5 +3,38 @@
+  <li>Merging across multiple courses with different names? You can repeat the course name search and add sections steps before clicking the Go Merge button.</li>
+  <li>When merging multiple sections, admins may want to edit the course name.</li>
+  <li>Teachers and other course managers can request additional assistance by contacting <a href="mailto:4help@umich.edu">4help@umich.edu</a>.</li>
+</ol>
```
```suggestion
</ul>
```

melindakraft commented on docs/merge-sections.html:
```diff
@@ -3,5 +3,38 @@
+  </ol>
+<h6>More Tips for using Merge Sections</h6>
+<ol>
+  <li>Merging across multiple courses with different names? You can repeat the course name search and add sections steps before clicking the Go Merge button.</li>
```
```suggestion
  <li>Sections must be in the same term to merge them with tool.</li>
  <li>Merging across multiple courses with different names? You can repeat the course name search and add sections steps before clicking the Go Merge button.</li>
```

melindakraft commented on docs/add-um-users.html:
```diff
@@ -3,5 +3,52 @@
+  <li>You can add users from a CSV file.</li>
+  <li>You can create a section on the fly and add users to it.</li>
+  <li>You can add users to multiple preexisting sections at one time.</li>
+  <li>The process does not send email notifications to new course members like the built-in Canvas People tool.</li>
```
```suggestion
  <li>Prevent Canvas course invitation messages and email notifications from being sent to new course members.</li>
```

melindakraft commented on docs/add-non-um-users.html:
```diff
@@ -3,5 +3,37 @@
+<ol>
+  <li>Click the Add Non U-M Users button on landing page.</li>
+  <li>Choose the &#8220;Add multiple users by uploading a CSV&#8221; option and click Select.</li>
+  <li>Pick a course section you would like to enroll users, and click Select.</li>
```
```suggestion
  <li>Click on the course section you would like to add the external users to, then click Select.</li>
```

melindakraft commented on docs/gradebook-thirdparty.html:
```diff
@@ -3,5 +3,26 @@
+  <li>Root Account Administrator</li>
+  <li>Sub-Account Administrator</li>
+  <li>Support Consultant</li>
+  <li>Teachers and other course managers who normally have access to the People tool</li>
```
```suggestion
  <li>Teachers and other course managers who have access to the course gradebook features. </li>
```

melindakraft commented on docs/create-sections.html:
```diff
@@ -3,5 +3,28 @@
+</ul>
+<h2 class="h5">How to Create Sections</h2>
+<ol>
+  <li>Click the Create Sections button on landing page.</li>
```
```suggestion
  <li>Click the Create Sections button on the landing page.</li>
```

melindakraft commented on docs/merge-sections.html:
```diff
@@ -3,5 +3,38 @@
+</ul>
+<h2 class="h5">How to Merge Sections</h2>
+<ol>
+  <li>In Canvas, go to the course that you want to merge sections into.</li>
```
```suggestion
  <li>Be sure you are in the Canvas course that you want to merge sections into.</li>
```

melindakraft commented on docs/merge-sections.html:
```diff
@@ -3,5 +3,38 @@
+<h2 class="h5">How to Merge Sections</h2>
+<ol>
+  <li>In Canvas, go to the course that you want to merge sections into.</li>
+  <li>Click the Merge Sections button on landing page.</li>
```
```suggestion
  <li>Click the Merge Sections button on the Course Manager landing page.</li>
```

melindakraft commented on docs/add-non-um-users.html:
```diff
@@ -3,5 +3,37 @@
+<p>External users can be added one at time by typing in the user&apos;s email address and other information, or in bulk by using a CSV file.</p>
+<h3 class="h6">Adding External Users One at a Time</h3>
+<ol>
+  <li>Click the Add Non U-M Users button on landing page.</li>
```
```suggestion
  <li>Click the Add Non U-M Users button on the Course Manager landing page.</li>
```

melindakraft commented on docs/add-um-users.html:
```diff
@@ -3,5 +3,52 @@
+<h2 class="h5">How to Add U-M Users to a Single Section</h2>
+<p>This feature allows administrators to select an existing section or create a new section to accept the users.</p>
+<ol>
+  <li>Click the Add U-M users button on landing page.</li>
```
```suggestion
  <li>Click the Add U-M users button on the Course Manager landing page.</li>
```

melindakraft commented on docs/add-um-users.html:
```diff
@@ -3,5 +3,52 @@
+  A list of course section IDs is included with this tool to help with CSV file preparation. 
+</p>
+<ol>
+  <li>Click the Add U-M users button on landing page.</li>
```
```suggestion
  <li>Click the Add U-M users button on the Canvas Course Manager landing page.</li>
```

melindakraft commented on docs/gradebook-thirdparty.html:
```diff
@@ -3,5 +3,26 @@
+</ul>
+<h2 class="h5">How to Use Format Third-Party Gradebook</h2>
+<ol>
+  <li>Click the Format Third‑Party Gradebook button on landing page.</li>
```
```suggestion
  <li>Click the Format Third‑Party Gradebook button on the Canvas Course Manager landing page.</li>
```

melindakraft commented on docs/index.html:
```diff
@@ -3,8 +3,25 @@
+  Like all third party tools, the first time you access Course Manager you will be prompted to authorize the tool. 
+  You can revoke your authorization in your Canvas Account settings. Updates to the Course Manager code may require you to reauthorize the tool. 
+</p>
+<h2 class="h5">Renaming Your Course</h2>
```
```suggestion
<h2 class="h5">Renaming A Course</h2>
```

melindakraft commented on docs/index.html:
```diff
@@ -3,8 +3,25 @@
+<ol>
+  <li>Authorized user can rename a course from the on landing page.</li>
+  <li>Launch the Canvas Course Manager from the course navigation menu. If you are already in the tool, click the Canvas Course Manager breadcrumb link at the top of the screen.</li>
+  <li>Click the Edit pencil icon next to the Course name, edit the course name as desired and click Save.</li>
```
```suggestion
  <li>On Canvas Course Manager landing page, click the Edit pencil icon next to the Course name, edit the course name as desired and click Save.</li>
```

ssciolla commented on docs/add-um-users.html:
```diff
@@ -3,5 +3,52 @@
+  <li>You can add users from a CSV file.</li>
+  <li>You can create a section on the fly and add users to it.</li>
+  <li>You can add users to multiple preexisting sections at one time.</li>
+  <li>Prevent Canvas course invitation messages and email notifications from being sent to new course members.</li>
```
```suggestion
  <li>It prevents Canvas course invitation messages and email notifications from being sent to new course members.</li>
```

ssciolla commented on docs/add-um-users.html:
```diff
@@ -3,5 +3,52 @@
+  <li>Root Account Administrator</li>
+  <li>Sub-Account Administrator</li>
+  <li>Support Consultant</li>
+  <li>Teachers and other course managers can request assistance by contacting <a href="mailto:4help@umich.edu">4help@umich.edu</a>.</li>
```
I still think this shouldn't be a `li`, instead, just a `p` outside the list, but @melindakraft can decide.

pushyamig replied:
I will keep it what I have since @melindakraft prefers that way.

ssciolla commented on docs/add-um-users.html:
```diff
@@ -3,5 +3,52 @@
+  Note that users added this way may receive a new course invitation message from Canvas.
+</p>
+<h2 class="h5">How to Add U-M Users to a Single Section</h2>
+<p>This feature allows administrators to select an existing section or create a new section to accept the users.</p>
```
```suggestion
<p>This option allows administrators to select an existing section or create a new section to which to enroll users.</p>
```

ssciolla commented on docs/add-non-um-users.html:
```diff
@@ -3,5 +3,37 @@
+<p>External users can be added one at time by typing in the user&apos;s email address and other information, or in bulk by using a CSV file.</p>
+<h3 class="h6">Adding External Users One at a Time</h3>
+<ol>
+  <li>Click the Add Non U-M Users button on the Course Manager landing page.</li>
```
```suggestion
  <li>Click the Add Non U-M Users button on the Canvas Course Manager landing page.</li>
```

ssciolla commented on docs/add-um-users.html:
```diff
@@ -3,5 +3,52 @@
+<p>This feature allows administrators to select an existing section or create a new section to accept the users.</p>
+<ol>
+  <li>Click the Add U-M users button on the Canvas Course Manager landing page.</li>
+  <li>Choose the &#8220;1 Section at a Time&#8221; option and click Select.</li>
```
```suggestion
  <li>Choose the &#8220;1 Section at a Time&#8221; option, and click Select.</li>
```

ssciolla commented on docs/add-um-users.html:
```diff
@@ -3,5 +3,52 @@
+  <li>Download the example CSV file, if needed.</li>
+  <li>Prepare and save your CSV file following the onscreen instructions for header and row content.</li>
+  <li>Drag and drop, or click the link to upload your saved CSV file.</li>
+  <li>CSV file data is presented back to you. Verify the results and click Submit.</li>
```
```suggestion
  <li>CSV file data is presented for you to review. Verify the results, and click Submit.</li>
```

ssciolla commented on docs/add-um-users.html:
```diff
@@ -3,5 +3,52 @@
+</ol>
+<h2 class="h5">How to Add U-M Users to Multiple Sections</h2>
+<p>
+  This feature allows administrators to add U-M users to more than one existing section at the same time. 
+  Users are not required to have the same role. New sections cannot be created with this tool.
```
```suggestion
  This option allows administrators to add U-M users to more than one existing section at the same time. 
  Users are not required to have the same role. New sections cannot be created with this option.
```

ssciolla commented on docs/add-um-users.html:
```diff
@@ -3,5 +3,52 @@
+</p>
+<p>
+  To use this feature you must include the Canvas section ID (not the section SIS ID) in your CSV file. 
+  A list of course section IDs is included with this tool to help with CSV file preparation. 
```
```suggestion
  A list of course section IDs is included to help with CSV file preparation. 
```

ssciolla commented on docs/add-um-users.html:
```diff
@@ -3,5 +3,52 @@
+  <li>Download the example CSV file, if needed.</li>
+  <li>
+    Prepare and save your CSV file following the onscreen instructions for header and row content. 
+    A list of course section IDs is presented on screen and a CSV version is available for download.</li>
```
```suggestion
    A list of course section IDs is presented on screen, and a CSV version is available for download.
  </li>
```

ssciolla commented on docs/create-sections.html:
```diff
@@ -3,5 +3,28 @@
+<h1 class="mt-5">Create Sections</h1>
+<p>
+  Some courses benefit from additional sections to organize students/participants for a variety of reasons. 
+  Course Manager makes new section creation quick and easy by uploading a simple CSV file.
```
```suggestion
  Canvas Course Manager makes new section creation quick and easy by allowing users to upload a simple CSV file.
```

ssciolla commented on docs/create-sections.html:
```diff
@@ -3,5 +3,28 @@
+  Course Manager makes new section creation quick and easy by uploading a simple CSV file.
+</p>
+<p>
+  <b>Tip: </b>Authorized users can add sections on the fly when adding U-M or non-U-M users with this tool. 
+  See the help documentation for these features for more information.
```
```suggestion
  <b>Tip: </b>Authorized users can add sections on the fly when adding U-M or non-U-M users. 
  See the help documentation for those features for more information.
```

ssciolla commented on docs/create-sections.html:
```diff
@@ -3,5 +3,28 @@
+  See the help documentation for these features for more information.
+</p>
+<h2 class="h5">Who Can Create Sections</h2>
+<p>Section management activities in this tool are limited to administrator roles:</p>
```
I think this needs to be "creation", since teachers can use "Merge Sections" which I would consider a section management activity.
```suggestion
<p>Section creation activities in this tool are limited to administrator roles:</p>
```

pushyamig replied:
I will stick with what I have before.

ssciolla replied:
This just feels inaccurate though.

melindakraft replied:
The is the section creation tool, there for "creation."

pushyamig replied:
@melindakraft if you like any SAm's Suggesting you can commit it now since That is is quickest thing to do.

ssciolla commented on docs/create-sections.html:
```diff
@@ -3,5 +3,28 @@
+  <li>Root Account Administrator</li>
+  <li>Sub-Account Administrator</li>
+  <li>Support Consultant</li>
+  <li>Teachers and other course managers can request assistance by contacting <a href="mailto:4help@umich.edu">4help@umich.edu</a>.</li>
```
Same deal, I'd make this a `p` tag outside the list.

pushyamig replied:
@melindakraft prefers this way. I will keep it in the list. Thanks!

ssciolla commented on docs/create-sections.html:
```diff
@@ -3,5 +3,28 @@
+</ul>
+<h2 class="h5">How to Create Sections</h2>
+<ol>
+  <li>Click the Create Sections button on the landing page.</li>
```
```suggestion
  <li>Click the Create Sections button on the Canvas Course Manager landing page.</li>
```

melindakraft replied:
```suggestion
  <li>Click the Create Sections button on the Canvas Course Manager landing page.</li>
```

ssciolla commented on docs/create-sections.html:
```diff
@@ -3,5 +3,28 @@
+<ol>
+  <li>Click the Create Sections button on the landing page.</li>
+  <li>Prepare and save a CSV file for upload (you can download an example file).</li>
+  <li>When your file is saved, drag and drop the file, or click the link Browse Your Files.</li>
```
```suggestion
  <li>When your file is saved, drag and drop the file, or click the &#8220;Browse Your Files&#8220; button.</li>
```

ssciolla commented on docs/create-sections.html:
```diff
@@ -3,5 +3,28 @@
+  <li>Click the Create Sections button on the landing page.</li>
+  <li>Prepare and save a CSV file for upload (you can download an example file).</li>
+  <li>When your file is saved, drag and drop the file, or click the link Browse Your Files.</li>
+  <li>Your file will display on the screen. Verify the results and click Submit. Click Cancel to Abort.</li>
```
```suggestion
  <li>CSV file data is presented for you to review. Verify the results, and click Submit. Click Cancel to abort.</li>
```

ssciolla reviewed (commented):
Releasing comments and suggestions for the first three features. Changes are mainly related to making things more consistent, logical, or clear. Looking good though.

ssciolla commented on docs/merge-sections.html:
```diff
@@ -3,5 +3,39 @@
+  <li>Be sure you are in the Canvas course that you want to merge sections into.</li>
+  <li>Click the Merge Sections button on the Canvas Course Manager landing page.</li>
+  <li>There are two panes in the Merge Sections tool. The right side pane shows sections already in the current course and will be updated to reflect new sections as you add them.</li>
+  <li>At the top of the left pane, enter the course name to merge in the search box.
+   <br> <b>Tip:</b> When working with large numbers of lab/discussion sections, search results are most efficient when you include the course number (e.g. STATS 250, not just STATS). </li>
```
```suggestion
  <li>
    At the top of the left pane, enter the course name to merge in the search box.
   <br><b>Tip:</b> When working with large numbers of lab/discussion sections, search results are most efficient when you include the course number (e.g. STATS 250, not just STATS).
 </li>
```

ssciolla replied:
I feel there's some information missing here about searching for uniqnames. Maybe you could say something like: "If you are a teacher, use the search box to filter the available sections. If you are admin, use the search box to search for sections by the associated course's name or by the instructor's uniqname. You can toggle between these options using the dropdown."

ssciolla commented on docs/merge-sections.html:
```diff
@@ -3,5 +3,39 @@
+  <li>There are two panes in the Merge Sections tool. The right side pane shows sections already in the current course and will be updated to reflect new sections as you add them.</li>
+  <li>At the top of the left pane, enter the course name to merge in the search box.
+   <br> <b>Tip:</b> When working with large numbers of lab/discussion sections, search results are most efficient when you include the course number (e.g. STATS 250, not just STATS). </li>
+  <li>Select the sections to merge by clicking on individual tiles or using the Select All box. 
+    <br> <b>Tip:</b> Use the Sort button to re-order the sections to merge by enrollments or alphabetical order.</li>
```
```suggestion
  <li>
    Select the sections to merge by clicking on individual tiles or using the Select All box. 
    <br><b>Tip:</b> Use the Sort button to re-order the sections to merge by number of enrollments or alphabetical order.
  </li>
```

ssciolla commented on docs/merge-sections.html:
```diff
@@ -3,5 +3,39 @@
+   <br> <b>Tip:</b> When working with large numbers of lab/discussion sections, search results are most efficient when you include the course number (e.g. STATS 250, not just STATS). </li>
+  <li>Select the sections to merge by clicking on individual tiles or using the Select All box. 
+    <br> <b>Tip:</b> Use the Sort button to re-order the sections to merge by enrollments or alphabetical order.</li>
+  <li>To remove sections from the right side pane, click the desired section tile, then click the undo button.</li>
```
I think there might be a missing step here; doesn't the user need to click "Add"?

ssciolla replied:
```suggestion
  <li>To remove sections from the right side pane, click the desired section tile, then click the Undo button.</li>
```

ssciolla commented on docs/merge-sections.html:
```diff
@@ -3,5 +3,39 @@
+  </ol>
+<h2 class="h6">More Tips for using Merge Sections</h6>
+<ul>
+  <li>Sections must be in the same term to merge them with tool.</li>
```
```suggestion
  <li>Sections must be in the same term to merge them using this tool.</li>
```

pushyamig commented:
@ssciolla I have accepted suggestion with punctuation in the sentence, But other suggestions with improved languages I will stick with what @melindakraft have. I appreciate your feedback!

ssciolla reviewed (approved):
Okay, I'm releasing the rest of my comments for the other documents.

@melindakraft, I understand if you don't want to incorporate any my text changes, but I am including them here in case you find them valuable. Otherwise just click "Resolve" and ignore them.

@pushyamig, I think the structure and formatting looks fine. I suggested a couple places where you could up longer lines morel.

pushyamig commented:
> @pushyamig, I think the structure and formatting looks fine. I suggested a couple places where you could up longer lines more.

I took them all, I was just breaking then when ended with a period. But I see you added a break between sentences good for clarity. 

@melindakraft I kept the sentence structure as the way you edited this morning. The only thing I changed today is HTML line spacing for code readability, I just changed minor punctuation suggestion which shouldn't be a big deal. `so no need for you to review from that`.
Normally if reviewers are happy with the code they show their consent by clicking the approve button that under `Files changed` tab ->  `Review Changes` -> Approve (you may also leave some final comments)

melindakraft commented on docs/gradebook-canvas.html:
```diff
@@ -3,5 +3,19 @@
+  <li>Click the Format Canvas Gradebook button on landing page.</li>
+  <li>Follow the instructions from the Requirement section on the “Upload your CSV File” page.</li>
+  <li>Drag and drop, or click the link to upload your exported gradebook CSV file.</li>
+  <li>The tool will convert your file to a Faculty Center compatible format and download it back to your computer.</li>
```
On my computer it doesn't give me a review step - it just downloads the new file

ssciolla replied:
I just checked on beta, I'm seeing the file shown to me before I click Submit, which then downloads the file. You're seeing something different?

melindakraft commented on docs/add-non-um-users.html:
```diff
@@ -3,5 +3,37 @@
 title: Add Non-UM Users
```
```suggestion
title: Add Non U-M Users
```

pushyamig replied:
@melindakraft This is something talked about with @janelilr and UX team. This is how it should be shown

melindakraft commented on docs/add-non-um-users.html:
```diff
@@ -3,5 +3,37 @@
-<h1 class="mt-5">Add Non-UM Users Documentation</h1>
-<p>Follow documentation for this Feature</p>
\ No newline at end of file
+<h1 class="mt-5">Add Non-UM Users</h1>
```
```suggestion
<h1 class="mt-5">Add Non U-M Users</h1>
```

melindakraft replied:
changing heading so it is U-M to follow established conventions

pushyamig replied:
@melindakraft This is how it is show in CCM app, I will stick with what I have. 
![Screen Shot 2022-04-15 at 4 02 14 PM](https://user-images.githubusercontent.com/8579775/163626731-5e5b1ae7-07a7-4fbe-a1b6-ef29651de0d8.png)

melindakraft commented on docs/add-non-um-users.html:
```diff
@@ -3,5 +3,37 @@
+  External users (those without a U-M uniqname) can be added to Canvas courses with the Canvas Course Manager tool. 
+  New external users who have never accessed Canvas will receive information by email to help them select a login method. 
+</p>
+<h2 class="h5">Who Can Add Non-UM Users</h2>
```
```suggestion
<h2 class="h5">Who Can Add Non U-M Users</h2>
```

melindakraft commented on docs/add-non-um-users.html:
```diff
@@ -3,5 +3,37 @@
+</ol>
+<h3 class="h6">Adding Multiple External Users to a Single Section with a CSV File</h3>
+<ol>
+  <li>Click the Add Non U-M Users button on landing page.</li>
```
```suggestion
  <li>Click the Add Non U-M Users button on the Canvas Course Manager landing page.</li>
```

melindakraft commented on docs/gradebook-canvas.html:
```diff
@@ -3,5 +3,19 @@
+</ul>
+<h2 class="h5">How to Use Format Canvas Gradebook</h2>
+<ol>
+  <li>Click the Format Canvas Gradebook button on landing page.</li>
```
```suggestion
  <li>Click the Format Canvas Gradebook button on the Canvas Course Manager landing page.</li>
```

melindakraft commented on docs/create-sections.html:
```diff
@@ -3,5 +3,28 @@
+  <li>Root Account Administrator</li>
+  <li>Sub-Account Administrator</li>
+  <li>Support Consultant</li>
+  <li>Teachers and other course managers can request assistance by contacting <a href="mailto:4help@umich.edu">4help@umich.edu</a>.</li>
```
```suggestion
  <li>Teachers and other course managers: please request assistance by contacting <a href="mailto:4help@umich.edu">4help@umich.edu</a>.</li>
```

pushyamig commented on docs/add-non-um-users.html:
```diff
@@ -3,5 +3,37 @@
+<p>External users can be added one at time by typing in the user&apos;s email address and other information, or in bulk by using a CSV file.</p>
+<h3 class="h6">Adding External Users One at a Time</h3>
+<ol>
+  <li>Click the Add Non U-M Users button on the Course Manager landing page.</li>
```
```suggestion
  <li>Click the Add Non U-M Users button on the Canvas Course Manager landing page.</li>
```

pushyamig commented on docs/gradebook-thirdparty.html:
```diff
@@ -3,5 +3,26 @@
+<p>
+  This feature converts CSV files from third-party tools that are not integrated with Canvas to a new format that can be used to import grades into the Canvas Gradebook.
+  This tool can be used to trim the third-party data to selected course sections. To import the new CSV to the gradebook, see the 
+  <a href="https://community.canvaslms.com/docs/DOC-12786-415255013" target="_blank">Canvas gradebook import function.</a>.
```
```suggestion
  <a href="https://community.canvaslms.com/docs/DOC-12786-415255013" target="_blank">Canvas gradebook import function.</a>
```
